src/core/optimization_container.jl

```diff
@@ -626,6 +626,7 @@ function get_variable(
     ::Type{U},
     meta::String = CONTAINER_KEY_EMPTY_META,
 ) where {T <: VariableType, U <: Union{PSY.Component, PSY.System}}
+    meta = remove_delimiter!(meta)
```

Comment:
The call to this function should be when we pass the name to the `add_container` call, not here.

Reply:
yup, I should have caught that